Library/Homebrew/cmd/tap.rb

```diff
@@ -66,7 +74,7 @@ def each_tap
   def tap_args(tap_name=ARGV.named.first)
     tap_name =~ HOMEBREW_TAP_ARGS_REGEX
     raise "Invalid tap name" unless $1 && $3
-    [$1, $3]
+    [$1, $3, ARGV.named[1]]
```

Comment:
`ARGV.named[1]` doesn't belong to this method. I mean you should fix the syntax problem in this way:

```ruby
user, repo = tap_args
clone_target  = ARGV.named[1]
opoo "Already tapped!" unless install_tap(user, repo, clone_target)
```